modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -321,9 +321,20 @@
  * A <b>shearing</b> transformation, sometimes called a skew, effectively
  * rotates one axis so that the x and y axes are no longer perpendicular.
  * <p>
- * Multiple transformations may be applied to a node by specifying an ordered
- * chain of transforms.  The order in which the transforms are applied is
- * defined by the ObservableList specified in the {@link #getTransforms transforms} variable.
+ * Multiple transformations may be applied to a node. Custom transforms are applied using the
+ * {@link #getTransforms transforms} list. Predefined transforms are applied using the properties specified below.
+ * The matrices that represent the transforms are multiplied in this order:
+ * <ol>
+ * <li> Layout ({@link #layoutXProperty layoutX}), {@link #layoutYProperty layoutY} and translate
```

Comment:
The closing paren should be after `layoutY`:

    * <li> Layout ({@link #layoutXProperty layoutX}, {@link #layoutYProperty layoutY}) and ...

Also, can you add `</li>` after each item?